src/main/deploy/pathplanner/New Path.path

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Should delete or rename to be more descriptive.

Reply:
i have no idea when i made this but its not needed so deleted